### DISCUSSION TITLE
MySQL preview - is this happening or avoid it

### DISCUSSION THREAD

**Discussion by snozwoz** [hasura/graphql-engine#8342] [General]
I am looking for an API solution for MySQL. 

We have a React/NextJs frontend and want to connect to data in an existing MySQL DB. Prefer a no-code solution. 

Not many options out there. Looked at AWS AppSync but that needs Aurora serverless and v2 has been in beta forever. EdgeDb looks interesting, but only if you have PostgreSQL, then found Hasura.

Hasura looks perfect, but again for Postgresql. Support for MySQL looks like it has been 'preview' for years. So should I avoid it? What is the roadmap for MySQL?

Thanks A

**Comment by codek** [comment]
I just tried the preview image, which is pretty old. Not sure what version it's based on.

I just got a malformed database URL parameter (possibly due to funky chars in the password?) and have given up for now.
